modules/tinymce/src/core/main/ts/api/OptionTypes.ts

```diff
@@ -136,6 +136,7 @@ interface BaseEditorOptions {
   invalid_elements?: string;
   invalid_styles?: string | Record<string, string>;
   keep_styles?: boolean;
+  newline_behavior?: string;
```

Comment:
Maybe move this so it's in alphabetical order? It was correct before we renamed it.

Reply:
oops.

Reply:
Something else I forgot to mention when reviewing the other day sorry. But should this be `'block' | 'linebreak' | 'invert' | 'default'` instead of `string`? As we don't allow any string here and it has to be a specific value.

Reply:
yes it should, I'll add it to the updates I have in progress